src/test/java/UhrzeitenTest.java

```diff
@@ -18,23 +17,10 @@ public void assembleAllValidTimes() {
 		assertThat(times.size(), not(2000));
 	}
 
-	@Test(expected = IndexOutOfBoundsException.class)
+	@Test(expectedExceptions = IndexOutOfBoundsException.class)
 	public void testIndexOutOfBoundsException() {
 		Uhrzeiten uhrzeiten = new Uhrzeiten();
 		List<String> times = uhrzeiten.assembleAllValidTimes();
 		times.get(-1);
 	}
-
-	@Test
```

Comment:
my old second test... as you probably remember we integrated a new one instead of this one in our last lesson already but when you made the pull request, my git repo wasn't up to date yet (lesson learned: always keep your git repo up-to-date). So can I do all these changes without endangering my project? not sure how it works with these conflicts...